lib/src/types.rs

```diff
@@ -40,7 +40,7 @@ impl Default for Class {
 pub struct FieldDecl {
     pub field_type: Type,
     pub name: String,
-    pub val: Option<String>,
+    pub val: Option<String>, // @Decide: Should probably Option<Expr> instead
```

Comment:
Agreed

Reply:
Maybe we should remove the value field, as `FieldDecl` is only a declaration.